RecoHI/Configuration/python/Reconstruction_HI_cff.py

```diff
@@ -35,6 +35,25 @@
 
 from RecoLocalCalo.Configuration.hcalGlobalReco_cff import *
 
+#post PF egamma stuff
+from RecoHI.HiEgammaAlgos.HiEgammaPostPF_cff import *
+
+from RecoHI.HiJetAlgos.HiRecoPFJets_cff import *
+PFTowers.src = cms.InputTag("particleFlow")
+akCs3PFJets.src = cms.InputTag("particleFlow")
+akCs4PFJets.src = cms.InputTag("particleFlow")
+kt4PFJetsForRho.src = cms.InputTag("particleFlow")
```

Comment:
why is this all changed here instead of the places where the modules are defined